integration/rpc_test.go

```diff
@@ -120,13 +120,16 @@ func TestEventsRPC(t *testing.T) {
 			buildEntries++
 		case *proto.Event_DeployEvent:
 			deployEntries++
+		case *proto.Event_DevLoopEvent:
+			devLoopEntries++
 		default:
```

Comment:
Please add a `t.Logf` here to we can catch possible flakes #4044 more quickly. (Or maybe a `t.Errorf` since they should be factored in?)

Reply:
done!